frontend/src/views/Character.vue

```diff
@@ -0,0 +1,522 @@
+
+<template>
+  <div class="p-5 mb-5 background-image">
```

Comment:
The margin bottom here needs removing. It's applying to the background image stopping it being the full screen